src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,21 @@
+package duke;
+
+/**
+ * A dedicated Exception class for Duke program.
+ *
+ */
+public class DukeException extends Exception {
```

Comment:
for the DukeException however i believe it would be better if you could create individual exceptions for each exception case as it would make the exception handling structure better! right now a DukeException being thrown will not let us know anything unless we read the message that is also thrown, but i think the exception itself should give some information as well (sorry for the repeat comment I had not put it in the PR review section properly)